app/services/create_github_repo_service/broadcaster.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class CreateGitHubRepoService
+  class Broadcaster
+    def self.call(entity, message, message_type, repo_url = nil)
+      new(entity, message, message_type, repo_url).call
```

Comment:
The fact that we're initializing an instance and then immediately throwing it a way is a smell that we should probably get rid of the instance methods here and just use class methods. I don't see any places in the codebase where we're relying on `Broadcaster` holding any state.

Reply:
I agree, but can we change it have an object with state and re-use it for wherever required something like:
```
broadcaster = Broadcaster.new(entity)
# for message
broadcaster.report_message(:repository_creation_complete, repo_url)
# for error
broadcaster.report_error(error_message)
```

Reply:
We could, but I don't see anywhere in the app where we're actually re-using a Broadcaster instance. As is, it's just unnecessary overhead we go through in Ruby every time we want to broadcast a message.